docs/porting/psa/spm.md

```diff
@@ -250,11 +250,21 @@ Processor access    |Secure RAM        |Secure FLASH|Nonsecure RAM      |Nonsecu
 
 ### Testing
 
-Arm provides a list of tests to make sure the HAL functions are implemented according to requirements, and the porting is done correctly.
+Arm provides a list of tests to check that the HAL functions are implemented according to requirements, and the porting is done correctly.
 
 After finalizing the porting, execute the following tests:
 
-- **tests-psa-spm_smoke:** This test will make sure that the porting of the mailbox mechanism (for dual core systems) is successful.
-- **tests-mbed_hal-spm:** This test will make sure the porting of the memory protection (*spm_hal_memory_protection_init()* implementation) makes the correct partitioning between secure RAM/Flash and nonsecure RAM/Flash.
+- **tests-psa-spm_smoke:** This test will check that the porting of the mailbox mechanism (for dual core systems) is successful.
+- **tests-mbed_hal-spm:** This test will check the porting of the memory protection (*spm_hal_memory_protection_init()* implementation) makes the correct partitioning between secure RAM/Flash and nonsecure RAM/Flash.
 
-We recommended you leave the memory protection part (*spm_hal_memory_protection_init()* implementation) to the end of the porting. First, implement and test other HAL functions. After these tests pass, implement *spm_hal_memory_protection_init()*, and run the entire test suite again, including the memory protection related tests.
+We recommended you leave the memory protection part (`spm_hal_memory_protection_init()` implementation) to the end of the porting. First, implement and test other HAL functions. After these tests pass, implement `spm_hal_memory_protection_init()`, and run the entire test suite again, including the memory protection related tests.
+
+This example shows how to run SPM tests for a PSA target with a nonsecure core:
+
+```
+mbed test -t GCC_ARM -m SPM_NONSECURE_CORE_PSA -n mbed-os-tests-psa-spm_smoke -v
+```
+

```

Comment:
There are two lines of code. What's the difference between them? Needs to be called out.

Reply:
These are two separate test commands. It's ok to combine them within the same code context if you prefer.